src/Illuminate/Collections/Arr.php

```diff
@@ -951,4 +951,35 @@ public static function wrap($value)
 
         return is_array($value) ? $value : [$value];
     }
+
+    public static function toFilled(array $arr, $value, int $start = 0, int $end = null): array
```

Comment:
First, docblock is missing.

I'd rename the `$arr` argument to `$array`. In 40 out of 43 public methods, in this class, the array argument is named `$array`, the exception being: 

- `Arr::accessible($value)` which checks if the `$value` is an array
- `Array::wrap($value)` which ensure `$value` is an array, and 
- `Arr::crossJoin(...$arrays)` which receives an array of arrays

Only in 4 public methods, the `$array` argument is typed. I'd say to keep yours typed, but just want to let you know.

In all public methods, regarding other arguments, only those which are a `callable` are typed. I'd say to move the `int` type hint to the missing docblock.

No method has a return type. Add it only to the missing docblock.

Reply:
It makes sense indeed. Renamed.